### PR TITLE
doc: fix typos

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

File: docs/Bootloaders.md [hunk 1]
```diff
@@ -382,13 +382,14 @@ make flash FLASH_DEVICE=/dev/ttyACM0
...
 not available, so it may be done by setting pin PA2 low if you flashed
-"hid_btt_skr_mini_e3.bin".  This pin is labeld "TX0" on the TFT header in
+"hid_btt_skr_mini_e3.bin".  This pin is labeled "TX0" on the TFT header in
 the SKR Mini E3's "PIN" document. There is a ground pin next to PA2
 which you can use to pull PA2 low.
 
 ### STM32F103/STM32F072 with MSC bootloader
 
-The [MSC bootloader](https://github.com/Telekatz/MSC-stm32f103-bootloader) is a driverless bootloader capable of flashing over USB.
+The [MSC bootloader](https://github.com/Telekatz/MSC-stm32f103-bootloader) is a
```
Comment (qwewer0) [inline docs/Bootloaders.md:391]: This was fine with 75 line.

Comment (thijstriemstra) [reply docs/Bootloaders.md]: @qwewer0 thanks for review, restored the line.

**Comment by JamesH1978** [comment]
Please refer to https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md and make sure you have included a signed-off-by line.

Thanks
James

**Comment by thijstriemstra** [comment]
Added a commit with the signed-off-by line; when squash committing this PR it should be included in the commit message..

**Review by qwewer0 — COMMENTED** [review]
The typo changes look good, but I didn't check for line length.

**Comment by thijstriemstra** [comment]
thanks for the review, please let me know if there's anything else I can do here.

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**